### ISSUE TITLE
Upstreaming the celluloid snap

### ISSUE THREAD

**Issue by soumyaDghosh** [celluloid-player/celluloid#965]
Hi @gnome-mpv it's great to see your enthusiasm regarding the snap package. It'd be great if the snap can be served from your repo and the snap to be published in the snap store under your name. I'm ready to help you out in maintaining it. The current maintainer isn't maintaining the snap properly, neither he seems to be co-operative. For context:

https://codeberg.org/sameersharma2006/celluloid-snap/issues/2
https://codeberg.org/sameersharma2006/celluloid-snap/issues/1

We all want to improve the quality of snaps, and a daily user of your snap, I would really like to have it properly tested and served and not in a half baked way.

**Comment by gnome-mpv** [comment]
Hi @soumyaDghosh. If you think there are problems with the Snap and you know how to fix those problems, you should submit a pull request to the repo. I'm sure @sameersharma2006 would be willing to review the pull request and merge it if it's reasonable.

**Comment by sameersharma2006** [comment]
Well sad, that workplace ethics no longer exists.
Is not transferring maintainership to you & not agreeing to your requests equates to being non-cooperative ?

I won't fuel this up, as i have had similar experience in the past with you, but i will surely say this is not  good way to deal with things.
Hoping something better from you in the future.

Regards,
Snap Maintainer.

**Comment by sameersharma2006** [comment]
> Hi @soumyaDghosh. If you think there are problems with the Snap and you know how to fix those problems, you should submit a pull request to the repo. I'm sure @sameersharma2006 would be willing to review the pull request and merge it if it's reasonable.

I am definitely open for contributions but name calling me & my work is not a good way & practice of dealing with things.
When you put so much of one's sacred time in one thing only to be shamed by others is disheartening & sad.

Looking forward for ethics based cooperation.

**Comment by soumyaDghosh** [comment]
Hi @gnome-mpv 
> If you think there are problems with the Snap and you know how to fix those problems, you should submit a pull request to the repo.

I don't agree with this, that "I should". Because there is a history of explaining someone something. And when someone needs the same explaination on the same problem again and again, i.e doesn't understand the codebase, or doesn't want to do enough testing, I don't think they should maintain a snap of that package in first place. This is what suggested to me by a Policy Reviewer of the Snap Store [in the open forum](https://forum.snapcraft.io/t/request-to-auto-connect-to-interfaces-for-snap-webcord/36431/7).

I don't think that when one explains something to someone, he should be replied with a label of `Contributions Welcome`, like it's an order for me to do so, which can be seen here in the threads also. I am not someone's employee, ordered to do something. I did try to have a polite conversations and that only went on me getting not so polite replies. I have already contributed to this snap in its initial days, when the snap didn't even build. So, I agree to disagree that I should not be questioned just because I think for the quality of apps in the store.
